js/src/components/contact-information/store-address-card.js

```diff
@@ -132,38 +108,61 @@ const StoreAddressCard = ( { showValidation = false } ) => {
 			.join( ', ' );
 
 		addressContent = (
-			<div>
+			<div className="gla-store-address-card__address">
 				<div>{ address }</div>
 				{ address2 && <div>{ address2 }</div> }
 				<div>{ rest }</div>
 			</div>
 		);
-	} else {
-		addressContent = <Spinner />;
 	}
 
+	const longDescription = (
+		<p>
+			{ createInterpolateElement(
+				__(
+					'We’re using your store address for Google verification. This information won’t be public. Edit in <link>WooCommerce settings</link> if needed and update to review the changes.',
+					'google-listings-and-ads'
+				),
+				{
+					link: settingsLink,
+				}
+			) }
+		</p>
+	);
+
+	const shortDescription = (
+		<p>
+			{ createInterpolateElement(
+				__(
+					'Your store address is required by Google for verification. This information won’t be public. Complete that in <link>WooCommerce settings</link> and update to review the changes.',
+					'google-listings-and-ads'
+				),
+				{
+					link: settingsLink,
+				}
+			) }
+		</p>
+	);
+
+	const detail = (
+		<>
+			{ addressContent }
+			{ ! isAddressFilled && (
```

Comment:
Doesn't look like it. Updated in 299fad753.

With this change, the only place using the new `useStoreAddressSynced()` hook is the `SetupAccounts` component ([ref](https://github.com/woocommerce/google-listings-and-ads/blob/30ac53a716cca086f4ac99ac28caf36339f9ee60/js/src/setup-mc/setup-stepper/setup-accounts/index.js#L94)) since we're no longer conditionally showing this component dependent on whether the address is synced, we could eliminate this hook altogether by updating the `useStoreAddress()` hook so it uses the `useGoogleMCAccount()` instead of directly calling `getGoogleMCContactInformation` action. That way we could bail early and avoid the error that is surfaced if there is no MC account connected.